Remotion/Validation/UnitTests/MetaValidation/Rules/System/MaximumLengthSystemPropertyMetaValidationRuleTest.cs

```diff
@@ -0,0 +1,112 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using System.Linq;
+using Moq;
+using NUnit.Framework;
+using Remotion.Reflection;
+using Remotion.Validation.Implementation;
+using Remotion.Validation.MetaValidation.Rules.System;
+using Remotion.Validation.UnitTests.TestDomain;
+using Remotion.Validation.Validators;
+
+namespace Remotion.Validation.UnitTests.MetaValidation.Rules.System
+{
+  [TestFixture]
+  public class MaximumLengthSystemPropertyMetaValidationRuleTest
+  {
+    private MaximumLengthSystemPropertyMetaValidationRule _rule;
+    private MaximumLengthValidator _maxLengthValidator1;
+    private MaximumLengthValidator _maxLengthValidator2;
+    private MaximumLengthValidator _maxLengthValidator3;
+
+    private Mock<IMaximumLengthValidator> _maxLengthValidatorMock1;
+    private Mock<IMaximumLengthValidator> _maxLengthValidatorMock2;
```

Comment:
please fix naming: Stub isntead of Mock sicne we won't do assertions, both here and in MinimumLengthSystemPropertyMetaValidationRuleTest